src/gtest/test_rpc.cpp

```diff
@@ -1,33 +1,450 @@
 // Copyright (c) 2018-2022 The Pastel developers
 // Distributed under the MIT software license, see the accompanying
 // file COPYING or https://www.opensource.org/licenses/mit-license.php.
+#include <regex>
+
 #include <gtest/gtest.h>
-#include <univalue.h>
 
-#include <chain.h>
-#include <chainparams.h>
-#include <clientversion.h>
-#include <primitives/block.h>
 #include <rpc/server.h>
-#include <streams.h>
+#include <rpc/client.h>
+#include <rpc/register.h>
+#include <key_io.h>
+#include <netbase.h>
+#include <main.h>
 #include <utilstrencodings.h>
+#include <univalue.h>
```

Comment:
univalue.h, 3rdparty - can go up to the same place a gtest.h

Reply:
done thanks